.gitattributes

```diff
@@ -1,8 +1,12 @@
-/.github/ export-ignore
-/bin/ export-ignore
-/tests export-ignore
-/.gitattributes export-ignore
-/.gitignore export-ignore
-/package.xml.tpl export-ignore
-/phpdoc.ini.dist export-ignore
-/phpunit.xml.dist export-ignore
+/.github/           export-ignore
+/bin/               export-ignore
+/tests              export-ignore
+/.editorconfig      export-ignore
+/.gitattributes     export-ignore
+/.gitignore         export-ignore
+/CHANGELOG.md       export-ignore
+/CONTRIBUTING.md    export-ignore
+/package.xml.tpl    export-ignore
+/phpdoc.ini.dist    export-ignore
+/phpunit.dist.xml   export-ignore
```

Comment:
Hmm, I just realised .dist.xml is only supported since PHPUnit 10. :(

Reply:
Hmm, I'll move this part to a different PR... will be less things to argue about.